packages/plugins/Gfi/src/store/actions.ts

```diff
@@ -132,7 +132,7 @@ const actions: PolarActionTree<GfiState, GfiGetters> = {
           layer
             // @ts-expect-error | layers reaching this have a source
             .getSource()
-            .on('changefeature', debouncedVisibilityChangeIndicator)
+            .on('addfeature', debouncedVisibilityChangeIndicator)
```

Comment:
The purpose of this action (`debouncedVisibilityChangeIndicator`) is to update a computed value in another position. Namely, when not moving the map at all, changing the filter and then checking the WFS list would previously result in an old view. That view is now linked to this indicator.

It is not `'addfeature'` anymore since the mechanism to change visibility has been modified. Previously, all features were *modified* one after another. Now, the whole set is copied and re-added after cleaning the source. While counter-intuitive at first, this results in far less follow-up computations by the cluster layer (that would, else, on any `'changefeature'` completely re-calculate clustering and block the browser up to 30s).

Reply:
https://github.com/Dataport/polar/pull/42/commits/833e86f8155886dfb12ab5a37e93b58e52210ea4

Writing about this made me do another change. Now, features are no longer copied, which is possible by "unlinking" them from the source before all changes take place. Since no copies are made anymore, less unexpected behaviour should take place when referring to features by reference in the future.